packages/wrangler/templates/no-op-assets-worker.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * This Worker is used as a default entry-point for Assets-only
+ * Workers. It proxies the request directly on to the Asset Sever
+ * Worker service binding.
+ *
+ * In an Assets-only Workers world, we want to enable users
+ * to deploy a Worker with Assets without ever having to provide
+ * a User Worker.
+ *
+ * ```bash
+ * wrangler dev --experimental-assets dist
+ * wrangler deploy --experimental-assets dist
+ * ```
+ *
+ * ```toml
+ * name = "assets-only-worker"
+ * compatibility_date = "2024-01-01"
```

Comment:
```suggestion
 * compatibility_date = "2024-01-01"
 * experimental_assets = "dist"
```

Reply:
technically, `experimental_assets` doesn't need to be defined here since we won't be doing anything with it. This no-op Worker is meant to proxy any incoming request to ASW, and was as a temp solution until I figure out how to have miniflare instantiate ASW without an entry-point Worker. I expect this template to be removed in a following PR, once that bit is figured out, so leaving as is for now